mipengine/node/monetdb_interface/udfs.py

```diff
@@ -1,8 +1,13 @@
 from typing import List
 
-from mipengine.node.monetdb_interface.monet_db_facade import db_execute
+from mipengine.node.monetdb_interface.monet_db_facade import db_execute_query
+from mipengine.node.monetdb_interface.monet_db_facade import db_execute_udf
 
 
-def run_udf(udf_statements: List[str]):
+def create_udf_defenitions(udf_statements: List[str]):
     for udf_stmt in udf_statements:
-        db_execute(udf_stmt)
+        db_execute_query(udf_stmt)
+
+
+def execute_udf(udf_exec_stmt: str):
+    db_execute_udf(udf_exec_stmt)
```

Comment:
Keep the same interface and move the logic of splitting the `udf_definitions` and `udf_execution` in this method.